app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -118,7 +118,7 @@
     <label>{{t 'Links & Social Media'}}</label>
     {{widgets/forms/link-input
       hasLinkName=true
-      fixedName=true
+      fixedName=false
```

Comment:
What has to be done here is, instead of making the change here, 

The `disabled=fixedName` from https://github.com/fossasia/open-event-frontend/blob/development/app/templates/components/widgets/forms/link-input.hbs#L15 needs to goto https://github.com/fossasia/open-event-frontend/blob/development/app/templates/components/widgets/forms/link-input.hbs#L3